docker/sorry-cypress/SETUP.md

```diff
@@ -0,0 +1,24 @@
+# Setup Sorry Cypress
+
+1. Use the docker compose file (called `sorry-cypress.yml`) in this folder as base.
+2. Replace the `<insert-minio-access-key>` and `<insert-minio-secret>` with applicable values (e.g random). These values are needed to manage the minio instance. 
+3. Set a random key for the `ALLOWED_KEYS` (by replacing `<insert-cypress-key>`). This key is needed to authorize bamboo against the sorry-cypress dashboard. 
+4. Place all the necessary NGINX config files in their locations
+    1. Place the `nginx.conf` file from this folder into this path `files/nginx/nginx.conf`
+    2. Place a generated `.htpasswd` file into this path `files/nginx/.htpasswd`
```

Comment:
Please elaborate a bit on this file. How you create it and what is it for would be important to document. I'd assume that it is for the basic auth.

Reply:
Done, yes it is used for basic auth